Engine/Makefile-defs.linux

```diff
@@ -38,7 +38,7 @@ SDL2_CFLAGS = $(shell sdl2-config --cflags)
 SDL2_LIBS = $(shell sdl2-config --libs)
 
 CFLAGS += $(SDL2_CFLAGS)
-LIBS += $(SDL2_LIBS) -lSDL2_sound
+LIBS += $(SDL2_LIBS) -Wl,-Bstatic -lSDL2_sound -Wl,-Bdynamic 
```

Comment:
i don't think this is necessary. if -lSDL2_sound is in the linker command it will pick up whatever library version (static or shared) that's available.

Reply:
We never should want the dynamically linked SDL2_sound since it has no package available for it, this is also how the rules have been written on Debian packaging. I can remove the dynamic library for it from the Docker library but removing here will cause people that try to build this makefile to mimic the AGS package having to remind and remove from the Makefile. Default behavior of the Makefile should statically link with SDL2_sound.

Reply:
how about installing only the static library from the self-built sdl2_sound then ? that would remove the need for this hack, and allow other linux users to link with whatever library flavor they seem fit (some distros install only shared libs, in which case this would cause a build failure).